source/core.py

```diff
@@ -120,6 +120,24 @@ class NewNVDAInstance:
 	directory: Optional[str] = None
 
 
+def restartUnsafely():
+	"""Force a new copy to start, which is useful if NVDA is frozen."""
+	import subprocess
+	for paramToRemove in ("--disable-addons", "--debug-logging", "--ease-of-access"):

```

Comment:
I'd keep these... If you run one instance with `--debug-logging` wouldn't you expect the replacement after a crash to have debug logging?

`--ease-of-access` can maybe be dropped? You'll need to look at how it affects NVDA to be sure.

Reply:
One of the things I did in the last restart PR was to carry over the `log.level` value. This means that if the logging level is changed at runtime, the new process maintains it.

Reply:
Removing this will be a change of behaviour from 2020.4. I'm good to make these changes, but I think it would be safest to do this on `alpha`.

Reply:
I think it is safer to keep the command line args, I'm quite positive disabling addons and debug logging are important to maintain. I don't know about ease of access.

Reply:
Restarting with addon's enabled is less safe than restarting with disabled. If addon's were disabled before, it was likely intentional and no the cause of the crash

It's possible that the debug logging is the cause of the crash. IE if we are trying to log something that causes an exception. I think this would be rare, and the recourse of manually stating NVDA is not terrible. If debug logging arg is present, the person is likely trying to debug NVDA, and may be trying to debug what happens during the crash restart.